dvc/repo/init.py

```diff
@@ -82,6 +82,8 @@ def init(root_dir=os.curdir, no_scm=False, force=False, subdir=False):
         [config.files["repo"], dvcignore, proj.plots.templates.templates_dir]
     )
 
+    logger.info("\nInitialized DVC repository.\n")
```

Comment:
Let's not add a newline before this message.
```suggestion
    logger.info("Initialized DVC repository.\n")
```